.gitignore

```diff
@@ -10,3 +10,4 @@
 /vendor/
 /bin/
 /composer.phar
+.mysql
```

Comment:
This directory required to survive container restart and keep all data in MySQL.